databases/week3/db_homework_week3.sql

```diff
@@ -0,0 +1,339 @@
+    'Olena',
+    'cool@gmail.com'
+  ),
+  (
+    '8',
+    2,
+    '2021-05-08',
+    '+38097-999-48-50',
+    'Alen',
+    'alen@gmail.com'
+  ),
+  (
+    '1',
+    3,
+    '2021-03-08',
+    '+38097-999-48-50',
+    'Nick',
+    'nick@gmail.com'
+  ),
+INSERT INTO
+  review (
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+VALUES
+  (
+    'Excellent',
+    'taste is excellent',
+    '1',
+    '5',
+    '2021-03-08'
+  ),
+  (
+    'very good',
+    'very good food',
+    '2',
+    '4',
+    '2021-03-08'
+  ),
+  ('good', 'good taste', '3', '3', '2021-03-08');
+  
+------Queries-----
+  -----meal queries-----
+  ---Get all meals---
+SELECT
+  *
+FROM
+  meal;
+---Add a new meal---
+INSERT INTO
+  meal(
+    `title`,
+    `description`,
+    `location`,
+    `when`,
+    `max_reservations`,
+    `price`,
+    `created_date`
+  )
+values(
+    'pizza',
+    'Italian dish',
+    'Rome',
+    '2021-09-01 11:18:05',
+    '10',
+    '150.00',
+    '2020-08-25 10:10:10'
+  );
+---Get a meal with any id, fx 1---
+SELECT
+  *
+FROM
+  review
+WHERE
+  id = 1;
+---Update a meal with any id, fx 1.Update any attribute fx the title or multiple attributes---
+UPDATE
+  meal
+SET
+  title = 'cheese set',
+  descriptions = '4 kind of cheese'
+WHERE
+  id = 1;
+---Delete a meal with any id, fx 1---
+DELETE FROM
+  meal
+WHERE
+  id = 1;
+---Reservation---
+  ---Get all reservations---
+SELECT
+  *
+FROM
+  reservation;
+----Add a new reservation---
+INSERT INTO
+  reservation (
+    `id`,
+    `number_of_guests`,
+    `meal_id`,
+    `created_date`,
+    `contact_phonenumber`,
+    `contact_name`,
+    `contact_email`
+  )
+VALUES
+  (
+    '3',
+    5,
+    '2021-02-08',
+    '+38097-258-48-50',
+    'Jens',
+    'jens@gmail.com'
+  );
+---Get a reservation with any id, fx 1---
+SELECT
+  *
+FROM
+  reservation
+WHERE
+  id = 2;
+---Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes--
+UPDATE
+  reservation
+SET
+  number_of_guests = 4,
+WHERE
+  id = 3;
+---Delete a reservation with any id, fx 1---
+DELETE FROM
+  reservation
+WHERE
+  id = 2;
+---Review---
+  ---Get all reviews---
+SELECT
+  *
+FROM
+  review;
+---Add a new review---
+INSERT INTO
+  review (
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+VALUES
+  (
+    'Exlusive',
+    'specific taste',
+    '4',
+    '5',
+    '2021-03-08'
+  ),
+  ---Get a review with any id, fx 1---
+SELECT
+  *
+FROM
+  review
+WHERE
+  id = 3;
+---Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes---
+UPDATE
+  review
+SET
+  `title` = 'deliciously',
+  `created_date` = '2021-05-12'
+WHERE
+  id = 2;
+---Delete a review with any id, fx 1---
+DELETE FROM
+  review
+WHERE
+  id = 3;
+----Additional queries----
+  ---Get meals that has a price smaller than a specific price fx 90--
+SELECT
+  meal
+WHERE
+  price < 90
+ORDER BY
+  price;
```

Comment:
You are missing something in your query, you should write `SELECT * meal where price < 90 ORDER BY price;`

Reply:
I`ve done this in two rows above.
Thank you!